docs/api-docs/promotions/code-samples.md

```diff
@@ -0,0 +1,9528 @@
+# Promotion Code Samples
+
+## Brand
+
+<details>
+  <summary>Spend $X in brand X, get free shipping to X shipping zones</summary> 
```

Comment:
Some places in this doc use "and" instead of a comma. Not sure which is better, but we should be consistent. For example, if we use "and", this sentence would say "Spend $X in brand X and get free shipping to X shipping zones"